spec/support/router_helpers.rb

```
@@ -0,0 +1,5 @@
+```ruby
```

RuboCop (1.76.1)

[fatal] unexpected '`', expecting end-of-input
(Using Ruby 3.4 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)